tests/docker.zunit

```diff
@@ -253,3 +1090,19 @@
     assert ${lines[2]} same_as '5425b1e460b6'
     assert ${lines[3]} same_as 'f17f844b1558'
 }
+
+@test 'Testing post: docker-containers' {
+    input=(
+        '862e5e30b1a7  app           \"/bin/sh\"           12 days ago  Exited (0) 12 days ago        gifted_elion'
+        '61858546cc23  5425b1e460b6  \"/bin/sh\"           12 days ago  Exited (0) 12 days ago        wonderful_tesla'
+        '295e3ea0de41  archlinux     \"/bin/zsh --login\"  4 weeks ago  Up 1 second                   thankful_smyth'
+    )
```

Comment:
Fix indentation and quotation styles.
```suggestion
    input=(
        '862e5e30b1a7  app           "/bin/sh"           12 days ago  Exited (0) 12 days ago  gifted_elion'
        '61858546cc23  5425b1e460b6  "/bin/sh"           12 days ago  Exited (0) 12 days ago  wonderful_tesla'
        '295e3ea0de41  archlinux     "/bin/zsh --login"  4 weeks ago  Up 1 second             thankful_smyth'
    )
```

Reply:
This spaces are equivalent to `PORTS` column.

Reply:
Yeah I have noticed about that after commeting this so see the updated one.